app/src/main/java/com/cornellappdev/transit/ui/screens/HomeScreen.kt

```diff
@@ -191,4 +197,79 @@ fun HomeScreen(
         }
     }
 
+    val scaffoldState = rememberBottomSheetScaffoldState()
+    var editState by remember {
+        mutableStateOf(false)
+    }
+    var txt by remember {
+        mutableStateOf("Edit")
+    }
+    val data = placesResponse.toMutableList()
+
+    BottomSheetScaffold(
+        scaffoldState = scaffoldState,
+        sheetSwipeEnabled = true,
+        sheetContent = {
+
+            Column() {
+                Row(
+                    modifier = Modifier
+                        .padding(bottom = 20.dp)
+                        .fillMaxWidth()
+                        .padding(
+                            bottom = 20.dp,
+                            top = 5.dp,
+                            start = 20.dp,
+                            end = 20.dp
+                        ),
+                    verticalAlignment = Alignment.CenterVertically
+                ) {
+                    Text(
+                        text = "Favorites",
+                        fontWeight = FontWeight.Bold,
+                        fontSize = 20.sp
+                    )
+                    Spacer(modifier = Modifier.weight(1f))
+                    Text(
+                        text = txt,
+                        modifier = Modifier.clickable {
+                            editState = editState == false
+                            txt = if (editState) {
+                                "Done"
+                            } else {
+                                "Edit"
+                            }
+                        },
+                        color = TransitBlue,
+                        textAlign = TextAlign.Right,
+                        fontSize = 14.sp,
+                    )
+                }
+
+                LazyRow(modifier = Modifier.padding(bottom = 20.dp)) {
+                    items(data) {
+                        LocationItem(
+                            image = painterResource(id = R.drawable.location_icon),
+                            editImage = painterResource(id = R.drawable.location_icon_edit),
+                            label = it,
+                            sublabel = it,
+                            editing = editState,
+                            onClick = {}
+                        )
+                    }
+                    item {
+                        LocationItem(
+                            image = painterResource(id = R.drawable.ellipse),
+                            editImage = painterResource(id = R.drawable.add_icon),
+                            label = "Add",
+                            sublabel = "",
+                            editing = editState,
+                            onClick = {}
+                        )
+                    }
+                }
+            }
+        }
+    ) {
+    }
```

Comment:
Sorry for only just noting this, but we should pull this out into its own component. I can imagine the home page getting awfully cluttered a few months down the line, so let's pull this out into its own file with a Composable named and spec'd appropriately!

Reply:
^ Feel free to pull specifically the code starting from the `Column`. I think having the scaffolding explicitly in the home screen is still probably best.